Update README.md for 0.6 (#3491)
quickwit-oss/quickwit: README.md

</h4>
<br/>

> <b>Quickwit 0.6 has just been released! Read our [blog post](https://quickwit.io/blog/release-0.6) to learn about the latest powerful features!</b>

Reviewer: ```suggestion
<b>We just released Quickwit 0.6! Read the [blog post](https://quickwit.io/blog/release-0.6) to learn about the latest powerful features!</b>
```

<br/>

<video src="https://github.com/quickwit-oss/quickwit/assets/653704/8a480fae-89f2-437c-bfc4-e3e49a5674e9" controls="controls" style="max-width: 1200px;">
</video>

<br/>

### **Quickwit is the fastest search engine on cloud storage. It is a great fit for:**

- [Log management](https://quickwit.io/docs/log-management/overview)
- [Distributed traces](https://quickwit.io/docs/distributed-tracing/overview)
- Any immutable data: conversational data (emails, texts, messaging platforms) & event-based analytics
… and more!

# 💡 Features

- Full-text search and aggregation queries
- Elasticsearch query language support
- Sub-second search on cloud storage (Amazon S3, Azure Blob Storage, …)
- Decoupled compute and storage, stateless indexers & searchers

> - [Schemaless](https://quickwit.io/docs/guides/schemaless) or strict schema indexing & and analytics

Author: yes... I was too lazy on this. We can add a new line "Schemaless analytics"
